_posts/2024-06-26-API-endpoints.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+category: Fundamentals
+title: "API endpoints"
```

Comment:
Needed for capitalization, figured I'd standardize this and the list css across posts.